flexget/components/status/db.py

```diff
@@ -52,7 +52,12 @@ def last_execution_time(cls):
         )
 
     def to_dict(self):
-        return {'id': self.id, 'name': self.name, 'last_execution_time': self.last_execution_time}
+        return {
+            'id': self.id,
+            'name': self.name,
+            'last_execution_time': self.last_execution_time
+            and self.last_execution_time.astimezone(),
```

Comment:
Is this a new syntax though? `last_execution_time` should be a datetime? But won't the `and` make it a bool?

Reply:
`and` does not coerce the value to a bool. `a and b` is equivalent to if `a` is truthy return `b` else return `a`.

Reply:
I do think `self.last_execution_time.astimezone() if self.last_execution_time else None` is easier to read though.